pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -740,6 +740,25 @@ func registerTPCC(r registry.Registry) {
 		},
 	})
 
+	r.Add(registry.TestSpec{
+		Name:              "tpcc-nowait/nodes=5/w=1000",
+		Owner:             registry.OwnerTestEng,
+		Benchmark:         true,
+		Cluster:           r.MakeClusterSpec(6, spec.CPU(16), spec.WorkloadNode()),
+		CompatibleClouds:  registry.AllClouds,
```

Comment:
Recommending `registry.OnlyGCE` here. Or at least avoid Azure.

Reply:
I want GCE and AWS so I changed it to `AllExceptAzure`